emx-digital/emx-digital-htb.js

```diff
@@ -169,6 +169,12 @@ function BRealTimeHtb(configs) {
                     }
                 };
             }
+
+            if (ccpaStatus) {
+                __emxData.us_privacy = ccpaStatus;
```

Comment:
Hi @ncolletti 

ccpaStatus in this case would be an object containing the following info provided by the CMP:
   - version: A number indicating the U.S. Privacy spec version
   - uspString: The U.S. Privacy string as defined in the IAB CCPA Compliance Framework for Publishers and Technology Companies 
e.g:
{
  version: 1,
  uspString: "1YNN"
}

Reply:
Thanks -- latest commit is now targeting 'uspString'